aws/cf.go

```diff
@@ -167,6 +176,13 @@ type denyResp struct {
 	body        string
 }
 
+func (spec *stackSpec) httpTargetPort() uint {
+	if spec.loadbalancerType == LoadBalancerTypeNetwork && spec.nlbHTTPEnabled && spec.nlbHTTPTargetPort != 0 {
+		return spec.nlbHTTPTargetPort
+	}
+	return spec.targetPort
+}
```

Comment:
This could be extended to support second target port for ALB in the similar way:
```go
...
} else if spec.loadbalancerType == LoadBalancerTypeApplication && spec.albHTTPTargetPort != 0 {
    return spec.albHTTPTargetPort
}
return spec.targetPort
```

Reply:
The idea is to detect of two ports are used via
```go
if port := spec.httpTargetPort(); port != spec.targetPort {
...
}
```

Reply:
I have added `--alb-http-target-port`